src/analyses/variable-sensitivity/value_set_abstract_value.h

```diff
@@ -0,0 +1,27 @@
+/*******************************************************************\
+
+ Module: analyses variable-sensitivity variable-sensitivity-value-set
+
+ Author: Diffblue Ltd.
+
+\*******************************************************************/
+
+/// \file
+/// Value sets for primitives
+
+// NOLINTNEXTLINE(whitespace/line_length)
+#ifndef CPROVER_ANALYSES_VARIABLE_SENSITIVITY_VALUE_SET_ABSTRACT_VALUE_H
```

Comment:
`NOLINTNEXTLINE` seems unnecessary (here and elsewhere)

Reply:
cpplint complains about the line length without these.